lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1116,21 +1115,63 @@ export class UserAgentApplication {
    * @ignore
    * @hidden
    */
-  private loadFrame(urlNavigate: string, frameName: string): void {
+  private loadAndRemoveFrame(urlNavigate: string, frameName: string): void {
     // This trick overcomes iframe navigation in IE
     // IE does not load the page consistently in iframe
     this._logger.info("LoadFrame: " + frameName);
     const frameCheck = frameName;
+    const frameHandle = this.addAdalFrame(frameCheck);
+    // Check hash on iframe
+    // 100ms delay each check for 5 times. 500ms total delay before function times out.
+    frameHandle.onload = () => {
+      this.recursiveDelayCheckFrameHash(frameHandle, 5);
```

Comment:
maybe pull magic number up into a const IFRAME_HANDLE_RETRY_COUNT or something

Reply:
Yeah. const magic number is always better